apps/src/gamelab/Exporter.js

```diff
@@ -110,8 +110,28 @@ export default {
       rewriteAssetUrls(appAssets, exportCode)
     );
 
-    // Attempt to fetch applab-api.min.js if possible, but when running on non-production
-    // environments, fallback if we can't fetch that file to use applab-api.js:
+    // webpack-runtime must appear exactly once on any page containing webpack entries.
+    //
+    // Attempt to fetch webpack-runtime.min.js if possible, but when running on
+    // non-production environments, fallback if we can't fetch that file to use
+    // webpack-runtime.js:
+
+    const webpackRuntimeAsset = new $.Deferred();
+    download('/blockly/js/webpack-runtime.min.js' + cacheBust, 'text').then(
+      (data, success, jqXHR) =>
+        webpackRuntimeAsset.resolve([data, success, jqXHR]),
+      download('/blockly/js/webpack-runtime.js' + cacheBust, 'text').then(
+        (data, success, jqXHR) =>
+          webpackRuntimeAsset.resolve([data, success, jqXHR]),
+        () =>
+          webpackRuntimeAsset.reject(
+            new Error('failed to fetch webpack-runtime.js')
+          )
+      )
+    );
```

Comment:
Can we deduplicate this out of the App Lab and Game Lab exporters?

Reply:
Done.